packages/appstore/src/modules/traders-hub/traders-hub.scss

```diff
@@ -66,3 +66,11 @@
     border-bottom: 1px solid var(--button-primary-default);
     border-radius: 0;
 }
+
+.get-started-trading-banner-ct {
```

Comment:
Instead of adding margin , can we add margin to the child tag of '.get-started-trading-banner-ct' ? 
That will prevent spacing when clevertap banner is disabled.